### PR TITLE
Add Link to Letra Github Page Inside Options Menu

### PR DESCRIPTION
**Pull request by tbarland77**
Here's my PR for #48. I'm going to be surprised if I got everything in one go, but figured it would be good to at least start a conversation about the changes I have so far. I'll also explain my changes below:

- I noticed for the styles there was already a fair amount of styling defined in base.scss so the only real styling I added was nowrap to prevent the text from going to the next line and I added the has-text-centered class so the link text is centered with the text inside the button.

- For the component, I just added two new properties inside of the data object as that seemed to be the place to initialize values and I thought it would be best not to hard code the link or the text inside of the template.

I also had a few general questions. Is Letra currently set up to use a linter? I noticed ESLint is part of the project, but when I do npm run lint it tells me to generate a config file. Just wanted to double check so that I can fix any linter errors if ESLint is being used. My other question was in regards to unit testing. It does not look like anything has been setup yet in that regard. Are there plans to add Jasmine or Jest in the future?

Anyway, hope you don't mind the lengthy comment! I look forward to any feedback.

### CODE REVIEW

**Comment by tbarland77** [comment]
Updated my branch name to refer to the correct issue. :sweat_smile:

File: client/src/style/options.scss [hunk 1]
```diff
@@ -70,3 +70,7 @@
   margin-top: $spacing-s;
   transition: all 0.8s;
 }
+
```
Comment (jayehernandez) [inline client/src/style/options.scss:73]: Can we change the class name to `.nowrap`? This may be reused for other stuff!

**Comment by jayehernandez** [comment]
Thanks for doing this PR! So fast 💨 

Just added a comment to change the class name just in case we need to use it for some other thing, we won’t need to create a new class.

Okay with not hard coding the text and link as well, good call with that one!

Haven’t had time to setup the linting and testing for this repo yet, sorry about that! The files seem to be okay in the linting aspect though!

**Comment by tbarland77** [comment]
No problem. I was pretty eager to start working on this yesterday! 

Thanks for the feedback. Changing the class name to make it more reusable makes sense to me. I'll work on that change soon.

Okay, that's fine! I just wanted to make sure there wasn't something I was missing in regards to the linter. As for the unit tests is that something you would be interested in having added to the repo sooner? I've worked with Jasmine and Jest before so I would not mind possibly helping out in that regard.

File: client/src/style/options.scss (hunk 1 above)
Comment (tbarland77) [reply client/src/style/options.scss:73]: The class name should now be updated to .nowrap.

**Comment by jayehernandez** [comment]
PR looks good to go! Going to release a new version tomorrow. I’ll be sure to include this 🤩 This closes #48. Hope you learned a thing or two about Pug!

Also for the unit tests, open to more discussions about this. I honestly don’t know how to do unit tests for Vue, but willing to start on it for this project. I’ve heard good things about using Jest with Vue, maybe we can brainstorm on that in a separate issue? Let me know.

@all-contributors please add @tbarland77 for code 🥳

**Pull request merged**